libs/designer/src/lib/core/DesignerProvider.tsx

```diff
@@ -39,6 +40,15 @@ export const DesignerProvider = ({ locale = 'en', options, children }: DesignerP
   const azTheme = !isDarkMode ? AzureThemeLight : AzureThemeDark;
   const webTheme = !isDarkMode ? webLightTheme : webDarkTheme;
   const themeName = useMemo(() => (!isDarkMode ? 'light' : 'dark'), [isDarkMode]);
+  const onError = useMemo<OnIntlErrorFn>(
```

Comment:
can we make this 
```suggestion
  const onError = useCallback<OnIntlErrorFn>(
```

useCallback is similar to useMemo but is made specifically for this use case.

Reply:
You're right, I totally forgot that function existed. Will do.